packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCryptoSelect.tsx

```diff
@@ -56,40 +57,57 @@ export const CoinmarketFormInputCryptoSelect = <
         [buildCryptoOptions, supportedCryptoCurrencies, sendCryptoSelectValue],
     );
 
+    const handleSelectChange = (selectedCryptoId: string) => {
+        const findOption = options.find(
+            option => option.type === 'currency' && option.value === selectedCryptoId,
+        ) as CoinmarketCryptoSelectItemProps | undefined;
+
+        if (!findOption) return;
+
+        if (isCoinmarketExchangeOffers(context)) {
+            context.setValue(FORM_RECEIVE_CRYPTO_CURRENCY_SELECT, findOption);
+        } else {
+            context.setValue(FORM_CRYPTO_CURRENCY_SELECT, findOption);
+        }
+
+        setIsModalActive(false);
+    };
+
+    const getNetworks = () => {
+        const eth = getNetwork('eth');
+        const polygon = getNetwork('matic');
```

Comment:
There is #14093 which renames the matic to pol. Should we use pol here instead of matic?

Reply:
yes

Reply:
what is the purpose of getting here those 4 networks?

Reply:
To show tabs 
![image](https://github.com/user-attachments/assets/f0f445e9-08cd-4c27-ab0c-ce365cab992a)

Reply:
After rebase, I will change it. )

Reply:
It does not make any sense. In the table there is like 50 networks. Next to "all networks" we say some random number and then for some reason we show four random networks next to it.

![Screenshot 2024-09-27 at 16 10 50](https://github.com/user-attachments/assets/ff48e5de-9264-4e2e-975d-00824a52db7f)

Reply:
Next to `All network` is not a random number. It is the number of all available networks. Soon it would be nice to add some select as you showed to us in DMs.

Reply:
I like that there is `Base` but we do not offer filter for it :(

![Screenshot 2024-09-27 at 16 21 04](https://github.com/user-attachments/assets/e35ff5fa-de70-4acf-9228-ac7cb9851c28)

Reply:
I finally understood, what you meant. :D Maybe we should change `All networks` to something like `All token networks`? What do you think?

Reply:
Please use order from networksConfig. We will sort it before freeze